action.yaml

```
@@ -125,7 +137,7 @@ runs:
       id: package-cache
       with:
         path: /tmp/.ubuntu-pkgs
-        key: ${{ runner.os }}-${{ steps.find-lvh-cli.outputs.runner_os_id }}-pkgs-cilium-little-vm-helper-${{ steps.find-lvh-cli.outputs.dependency_list_sha }}
+        key: lvh-${{ runner.os }}-${{ steps.find-lvh-cli.outputs.runner_os_id }}-pkgs-cilium-little-vm-helper-${{ steps.find-lvh-cli.outputs.dependency_list_sha }}
```

maybe we can drop that part then?
```suggestion
        key: lvh-${{ runner.os }}-${{ steps.find-lvh-cli.outputs.runner_os_id }}-pkgs-${{ steps.find-lvh-cli.outputs.dependency_list_sha }}
```